### PR TITLE
Update list of required lnd ENV variables

### PR DESCRIPTION
**Pull request by jholton**
There's a few things going on here.
I'm afraid I have more questions than fixes with this PR. 😄 

First,
> You'll need to add some values to the `.env` inside the `server` sub-project.

but, there is no `.env` file in the `server` sub-project. [link](https://github.com/bmancini55/building-lightning-invoices/tree/main/server).

So, I created one and added the env values specified in the walkthrough:
- `LND_RPC_HOST`
- `LND_ADMIN_MACAROON_PATH`
- `LND_CERT_PATH`

But when I tried to run the `npm run script:create-invoice`, I got an error
> TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received undefined

The stacktrace led me to `server/src/Options.ts` where I see it's trying to read [`LND_INVOICE_MACAROON_PATH`](https://github.com/bmancini55/building-lightning-invoices/blob/689c6696e7d6ea0ae15591eeb16622c9de0e0c2d/server/src/Options.ts#L18) and [`LND_READONLY_MACAROON_PATH`](https://github.com/bmancini55/building-lightning-invoices/blob/689c6696e7d6ea0ae15591eeb16622c9de0e0c2d/server/src/Options.ts#L19), which were not specified in the instructions (hence this PR).

I also noticed that `server/src/Options.ts` also sets [`PORT`](https://github.com/bmancini55/building-lightning-invoices/blob/689c6696e7d6ea0ae15591eeb16622c9de0e0c2d/server/src/Options.ts#L13) and [`LND_REST_HOST`](https://github.com/bmancini55/building-lightning-invoices/blob/689c6696e7d6ea0ae15591eeb16622c9de0e0c2d/server/src/Options.ts#L14) 
(note that `LND_REST_HOST` is called [`LND_HOST` in `building-lightning-graph`](https://github.com/bmancini55/building-lightning-graph/blob/b7a692c1ed5ea524fa8b0a397b51204ef1f17cfa/server/.env#L6))

In summary, my outstanding questions:
- Should `.env` already exist?
- Should `LND_REST_HOST` be called `LND_REST_HOST` or `LND_HOST`?
- Are `LND_REST_HOST` and `PORT` necessary at all?  (I'm not done with the walkthrough yet)

### CODE REVIEW

**Comment by bmancini55** [comment]
Ah apologies for the confusion. Things changed a bit between the first and second project due to the change to using the GRPC API.  To answer your questions:

>     * Should `.env` already exist?

Yes, ideally `.env` should exist to make the tutorial easier. I will go ahead and make this change in the code repository shortly.

>     * Should `LND_REST_HOST` be called `LND_REST_HOST` or `LND_HOST`?

The REST API should be `LND_REST_HOST` to disambiguate between the GRPC host.

>     * Are `LND_REST_HOST` and `PORT` necessary at all?  (I'm not done with the walkthrough yet)

`LND_REST_HOST` is not actually necessary in the tutorial, but if someone wanted to explore the REST API usage they would need it. So it's probably ok to leave it in there.
`PORT` is used by the express application, so it is necessary.

Hopefully that clarifies a bit.

**Comment by jholton** [comment]
Thanks for the feedback.

I have:
- added a line about `LND_REST_HOST` being optional
- changed the list of env variables to match the `.env` file you added in the `building-lightning-invoices` repo.

**Review by bmancini55 — APPROVED** [review]
Awesome, thank you very much!
